torchci/pages/api/usage-log-aggregator/lambda.ts

```diff
@@ -0,0 +1,14 @@
+import * as restc from "typed-rest-client/RestClient";
+import { NextApiRequest, NextApiResponse } from "next";
+
+// TODO: Create a hostname and secure this API
```

Comment:
None of the API we are using have and API key, that's what I mean here including the one that accepts GitHub hooks (shhh!) and write to S3.  This API only reads the data from S3 (read-only), so I think I can leave the API key part later